backend/climateconnect_main/settings.py

```diff
@@ -247,11 +248,20 @@
 IDEA_MENTION_TEMPLATE_ID_DE = env('IDEA_MENTION_TEMPLATE_ID_DE')
 JOINED_IDEA_TEMPLATE = env('JOINED_IDEA_TEMPLATE')
 JOINED_IDEA_TEMPLATE_DE = env('JOINED_IDEA_TEMPLATE_DE')
+WEEKLY_RECOMMENDATIONS_EMAIL = env('WEEKLY_RECOMMENDATIONS_EMAIL')
+WEEKLY_RECOMMENDATIONS_EMAIL_DE = env('WEEKLY_RECOMMENDATIONS_EMAIL_DE')
 PROJECT_JOIN_REQUEST_TEMPLATE_ID = env('PROJECT_JOIN_REQUEST_TEMPLATE_ID')
 PROJECT_JOIN_REQUEST_TEMPLATE_ID_DE = env('PROJECT_JOIN_REQUEST_TEMPLATE_ID_DE')
 
+WEEKLY_RECOMMENDATIONS_EMAIL = env('WEEKLY_RECOMMENDATIONS_EMAIL')
+WEEKLY_RECOMMENDATIONS_EMAIL_DE = env('WEEKLY_RECOMMENDATIONS_EMAIL_DE')
+
+USER_CHUNK_SIZE = env('USER_CHUNK_SIZE', 100)
+
 
 FRONTEND_URL = env('FRONTEND_URL', '')
+BACKEND_URL = env('BACKEND_URL') if env('ENVIRONMENT') in ('development', 'test') else 'https://'+env(
+    'AZURE_ACCOUNT_NAME')+'.'+env('AZURE_HOST')+'/{}/'.format(env('AZURE_CONTAINER'))
```

Comment:
Why do you need this for?

Reply:
This is needed for getting the correct image path to the thumbnail of entities